src/Features/Core/Portable/RemoveUnnecessaryParentheses/AbstractRemoveUnnecessaryParenthesesDiagnosticAnalyzer.cs

```diff
@@ -47,7 +47,7 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
             var parenthesizedExpression = (TParenthesizedExpressionSyntax)context.Node;
 
             if (!CanRemoveParentheses(parenthesizedExpression, context.SemanticModel,
-                    out var precedenceKind, out bool clarifiesPrecedence))
+                    out var precedenceKind, out var clarifiesPrecedence))
```

Comment:
really? we even use var for bool in ide code?

Reply:
up to the dev. we allow var whenever legal if htat's what is preferred.